ios/build.sh

```diff
@@ -135,16 +142,15 @@ echo "Building KMEI..."
 rm -r $KMEI_BUILD_PATH/$CONFIG-universal 2>/dev/null
 xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-universal -configuration $CONFIG \
```

Comment:
Now that KeymanEngine is built along with Keyman, maybe put the building of the universal framework behind a flag since it's not always required.

Reply:
I guess my more general observation is that the script assumed previously that KeymanEngine is a dependency of Keyman, which is no longer the case in this script (it still is in Xcode). So some of the way the script is organized could be rethought (although that could be deferred for later).